ec/resources/rulesengine/v4policy.go

```diff
@@ -248,24 +259,54 @@ func standardizeMatchFeature(matchFeatureMap map[string]interface{}) {
 	}
 }
 
-func getDeployRequestData(d *schema.ResourceData, policyID int) *api.AddDeployRequest {
+func getDeployRequestData(
+	d *schema.ResourceData,
+	policyID int,
+) *api.AddDeployRequest {
 	return &api.AddDeployRequest{
 		Message:     "Auto-submitted policy",
 		PolicyID:    policyID,
 		Environment: d.Get("deploy_to").(string),
 	}
 }
 
-func addPolicy(policy string, isEmptyPolicy bool, d *schema.ResourceData, m interface{}) error {
+func getPolicy(
+	m interface{},
+	d *schema.ResourceData,
+) (map[string]interface{}, error) {
+	config := m.(**api.ClientConfig)
+	(*config).AccountNumber = d.Get("account_number").(string)
+	// Portal Type IDs: 1=MCC 2=PCC 3=WCC 4=WCC 5=opencdn
+	portalTypeID := d.Get("portaltypeid").(string)
+	customerUserID := d.Get("customeruserid").(string)
+	policyID, _ := strconv.Atoi(d.Id())
```

Comment:
Are we certain we wont have an error here?

Reply:
Reasonably certain since the value comes from the state file... wouldn't hurt to put in an error check though